Dte.Common/Content/CustomMessageEmail.cs

```diff
@@ -1,429 +1,48 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
 namespace Dte.Common.Content
 {
     public static class CustomMessageEmail
     {
-        // TODO - html file or S3, make the text replaceable so can replace for difference trigger sources.
-        public static string GetCustomMessageHtml()
+        private const string BodyPlaceholder = "###BODY_REPLACE###";
+        private const string EmailTemplateResourceSuffix = "EmailTemplate.html";
+        private static readonly Lazy<string> CachedTemplate = new Lazy<string>(LoadTemplate);
```

Comment:
The caching should be transparent, and is handled by `Lazy` don't make it obvious from the variable name.

It is just `EmailTemplate`.